### DISCUSSION TITLE
[Feature Request] Sleep Timer set by default

### DISCUSSION THREAD

**Discussion by starfetch** [PaulWoitaschek/Voice#1743] [Ideas]
It would be great to have a setting to turn on the sleep timer everytime you start playback by default. I'm trying to automate my sleep audiobook, and while I'm able to start playing it on Voice, I still have to manually start the timer. An optional default setting would therefore be great.

Thanks for all the work and the amazing app!

**Comment by Ansem93** [comment]
100% this!
That feature and shake to restart sleep timer is why I'm using an other app.
Just a note: the sleep timer should not be started if the device is connected to android auto

**Comment by happy-turtle** [comment]
I would love this as well. I know this feature from [antenna](https://github.com/AntennaPod/AntennaPod/blob/develop/core%2Fsrc%2Fmain%2Fjava%2Fde%2Fdanoeh%2Fantennapod%2Fcore%2Fservice%2Fplayback%2FPlaybackService.java#L854). I am open to work on it, but I don't have much experience with Kotlin.

**Comment by fbbb2** [comment]
I would love to see that too!

**Comment by pixel-fabian** [comment]
The request ist mentioned in https://github.com/PaulWoitaschek/Voice/discussions/2387#discussioncomment-9684350 as well. 

I use the feature from AntennaPod, where you can set specific hours (e.g. 10pm to 6am) and during that time the sleep timer will be turned on by default and would love to see that in Voice as well.

**Comment by happy-turtle** [comment]
It has been implemented with #2673, 221e96a8036c6dc57971b98c8b711183cab939d2 and later commits. So it will probably come with one of the next feature versions.
